.jenkins/pytorch/test.sh

```diff
@@ -508,6 +519,7 @@ if [[ "${BUILD_ENVIRONMENT}" == *backward* ]]; then
   # Do NOT add tests after bc check tests, see its comment.
 elif [[ "${BUILD_ENVIRONMENT}" == *xla* || "${JOB_BASE_NAME}" == *xla* ]]; then
   install_torchvision
+  build_xla
```

Comment:
No idea why were previously dedicating an entire job just to building xla when the base pytorch remained the same.